mappings/net/minecraft/client/particle/ParticleManager.mapping

```diff
@@ -2,14 +2,54 @@ CLASS dcp net/minecraft/client/particle/ParticleManager
 	FIELD a world Lbbp;
 	FIELD b PARTICLE_TEX Lqc;
 	FIELD c particleQueues [[Ljava/util/ArrayDeque;
+	FIELD d newEmitterParticles Ljava/util/Queue;
 	FIELD e textureManager Ldnp;
 	FIELD f random Ljava/util/Random;
+	FIELD g factories Lit/unimi/dsi/fastutil/ints/Int2ObjectMap;
 	FIELD h newParticles Ljava/util/Queue;
 	METHOD <init> (Lbbp;Ldnp;)V
 		ARG 1 world
+		ARG 2 textureManager
 	METHOD a tick ()V
+	METHOD a updateGroup (I)V
+		ARG 1 group
+	METHOD a renderUnlitParticles (Lagv;F)V
+		ARG 1 cameraEntity
+	METHOD a addEmitter (Lagv;Lft;)V
+		ARG 1 entity
+		ARG 2 parameters
+	METHOD a addEmitter (Lagv;Lft;I)V
+		ARG 1 entity
+		ARG 2 parameters
+		ARG 3 maxAge
 	METHOD a setWorld (Lbbp;)V
 	METHOD a addParticle (Ldco;)V
+	METHOD a addBlockBreakParticles (Let;Lbpm;)V
+		ARG 1 pos
+		ARG 2 state
+	METHOD a addBlockBreakingParticles (Let;Ley;)V
+	METHOD a addParticle (Lft;DDDDDD)Ldco;
+		ARG 1 parameters
+		ARG 2 x
+		ARG 4 y
+		ARG 6 z
+		ARG 8 xVelocity
```

Comment:
Seems to have a mix of `velocityX` and `xVelocity` -- should probably make this consistent.

Reply:
Now consistently velocityX/Y/Z